src/tabs/tabset.component.html

```diff
@@ -1,10 +1,9 @@
-<ul class="nav" [ngClass]="classMap" (click)="$event.preventDefault()">
+<ul class="nav" [ngClass]="classMap" (click)="$event.preventDefault()" role="tablist">
   <li *ngFor="let tabz of tabs" [ngClass]="['nav-item', tabz.customClass || '']"
       [class.active]="tabz.active" [class.disabled]="tabz.disabled">
-    <a href="javascript:void(0);" class="nav-link"
-       [attr.id]="tabz.id ? tabz.id + '-link' : ''"
-       [class.active]="tabz.active" [class.disabled]="tabz.disabled"
-       (click)="tabz.active = true">
+    <a href="javascript:void(0);" class="nav-link" [attr.id]="tabz.id ? tabz.id + '-link' : ''"
```

Comment:
no need for this condition `tabz.id ? tabz.id + '-link' : ''` since we have a default value for `id`, just `tabz.id + '-link'`